src/PhpCodeFixer.php

```diff
@@ -141,13 +141,25 @@ public function checkDir($dir) {
      * @param Report $report
      */
     protected function checkDirInternal($dir, Report $report) {
+        if (in_array(strtolower(basename($dir)), $this->excludeList, true)) {

```

Comment:
I think that's unnecessary condition

Reply:
Without that one you can't exclude top level folders. You can test it by excluding vendor or src within this repository.

Reply:
It's missing the continue statement as well though.